package/MDAnalysis/core/groups.py

```diff
@@ -1624,7 +1628,6 @@ def unwrap(self, compound='fragments', reference='com', inplace=True):
                 c = unique_atoms[mask]
                 positions[mask] = mdamath.make_whole(c, inplace=False)
                 # Apply reference shift if required:
-                if reference is not None:
                     if ref == 'com':
```

Comment:
Could you check the indentation here. From the diff it looks like you removed the `if` statement but did not remove the indentation on the block.